bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1016,7 +1022,7 @@ string BCStateTran::stateName(FetchingState fs) {
   }
 }
 
-std::ostream &operator<<(std::ostream &os, const BCStateTran::FetchingState fs) {
+inline std::ostream &operator<<(std::ostream &os, const BCStateTran::FetchingState fs) {
```

Comment:
Do we need to return os?

Reply:
Yes we must. If you don't do that you will not be able to have operator << concatenated:
std::cout << a << b << c;